ui/app/AppLayouts/Onboarding2/KeycardCreateProfileFlow.qml

```diff
@@ -16,23 +16,23 @@ SQUtils.QObject {
     required property int addKeyPairState
     required property int keycardPinInfoPageDelay
 
-    required property var seedWords
+    required property var getSeedWords
     required property var isSeedPhraseValid
 
     property bool displayKeycardPromoBanner
 
     signal loginWithKeycardRequested
     signal keycardFactoryResetRequested
-    signal keyPairTransferRequested
+    signal loadMnemonicRequested
     signal keycardPinCreated(string pin)
+    signal setPinFailed(string error)
```

Comment:
For async operations like adding key pair we have dedicated enum defined (`AddKeyPairState` in enums.h`) which is used for handling that ansyc operation. Similarly for syncing. Here the approach is different, UI emits `keycardPinCreated` and expects that at some point in the future `setPinFailed` will be called or some other thing will happen which means success (btw, how we may know that pin setting succeeded)?